### PR TITLE
Add Cert Secret Geneator to Health Patterns

### PR DESCRIPTION
**Pull request by ibmatc**
Signed-off-by: Adam T. Clark <atclark@us.ibm.com>

### CODE REVIEW

File: services/cert-secret-generator/chart/templates/role.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "cert-secret-generator.fullname" . }}-role
+  namespace: {{ .Release.Namespace }}
+rules:
+  - verbs:
+      - get
+      - list
```
Comment (ranum) [inline services/cert-secret-generator/chart/templates/role.yaml:9]: Did you want this list to be different?

File: services/cert-secret-generator/gen_certs.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Exit script on any error
+#set -e
+
+
+PASSWORD=integrati0n
```
Comment (ranum) [inline services/cert-secret-generator/gen_certs.sh:7]: Do we want to hardcode this value into the script? or maybe use some other init value?

**Review by ranum — APPROVED** [review]
LGTM

File: services/cert-secret-generator/gen_certs.sh (hunk 2 above)
Comment (ibmatc) [reply services/cert-secret-generator/gen_certs.sh:7]: Good point. I'll change it to "replace-me".  It shouldn't get used by our process as the helm chart-docker container chain will always pass in an argument that'll be used, but I'll genericize it for the container.

File: services/cert-secret-generator/chart/templates/role.yaml (hunk 1 above)
Comment (ibmatc) [reply services/cert-secret-generator/chart/templates/role.yaml]: Yeah, updated.  Must have lost the change in Git somewhere...

**Pull request merged**